rights-statements.ttl

```diff
@@ -28,202 +30,219 @@
 
 <collection-ic> a skos:Collection ;
```

Comment:
I guess I don't have a very strong feeling about it, but I do want to ensure that we're consistent.